frontend/packages/dev-console/src/components/add/AddCardItem.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { SimpleListItem, Title, Text } from '@patternfly/react-core';
+import { ResolvedExtension, AddAction } from '@console/dynamic-plugin-sdk';
+import { useTelemetry } from '@console/shared/src/hooks/useTelemetry';
+import { navigateTo, resolvedHref } from '../../utils/add-page-utils';
+import { useShowAddCardItemDetails } from '../../hooks/useShowAddCardItemDetails';
+import './AddCardItem.scss';
+
+type AddCardItemProps = {
+  action: ResolvedExtension<AddAction>;
+  namespace: string;
+};
+
+const AddCardItem: React.FC<AddCardItemProps> = ({
+  action: {
+    properties: { id, label, icon, href, description },
+  },
+  namespace,
+}) => {
+  const fireTelemetryEvent = useTelemetry();
+  const [showDetails] = useShowAddCardItemDetails();
+
+  const actionIcon = (): JSX.Element => {
+    if (typeof icon === 'string') {
+      return (
+        <img
+          className="odc-add-card-item__header__icon"
+          src={icon}
+          alt={label}
+          aria-hidden="true"
+        />
+      );
+    }
+    if (typeof icon !== 'string' && React.isValidElement(icon)) {
+      return (
+        <span className="odc-add-card-item__header__icon" aria-hidden="true">
+          {icon}
+        </span>
+      );
+    }
+    return null;
+  };
+
+  return (
+    <div className="odc-add-card-item">
```

Comment:
Please add a data-test-id here so that we can find this specific add card item.

```suggestion
    <div className="odc-add-card-item" data-test-id={id}>
```

Reply:
done